### PR TITLE
Incorporate feedback from the Ethical ML workshops

### PR DESCRIPTION
**Pull request by anssiko**
- Update Operationalization: Putting the Principles into Practice
- Add Register of Risks and Mitigations

***
<a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/pull/20.html" title="Last updated on Jun 2, 2022, 1:31 PM UTC (a272f56)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/webmachinelearning/ethical-webmachinelearning/20/25a8ba5...a272f56.html" title="Last updated on Jun 2, 2022, 1:31 PM UTC (a272f56)">Diff</a>

### CODE REVIEW

**Comment by anssiko** [comment]
Gentle ping, PTAL @dontcallmedom :-)

Some areas to focus on, mostly presentational:

- Proposals welcome on how to better represent Risks -> Possible Mitigations mapping. Maybe with some CSS tricks we could use a two-column layout for Risks and Possible Mitigations side-by-side (when screen estate allows)? Someone with proper CSS-fu would need to help with that. Or we could consider that future work.

- "This bullet intentionally left blank" in Possible Mitigations. I was thinking of turning these empty bullets into links that open a new GH issue to encourage contributions.

Comments:

- I'd prefer to keep the source format as simple as possible and let CSS handle presentational aspects, if possible. Complex tables are tedious to edit.

- To help editing the lists and correlate Risks with Possible Mitigations in source, maybe instead of:
```
1.

1.

1.
```

We should use:

```
1.

2.

3.
```

File: index.bs
Comment (dontcallmedom) [inline index.bs]:
```suggestion
To provide a more structured approach to this, we have developed a workshop format that can be adapted and used to help identify such risks & mitigations.
```

File: index.bs
Comment (dontcallmedom) [inline index.bs]:
```suggestion
The workshop is articulated around two documents that are expected to be completed interactively by participants:
```

File: index.bs
Comment (dontcallmedom) [inline index.bs]: we should probably post a PDF copy of these documents somewhere if only for archival sake.

File: index.bs
Comment (dontcallmedom) [reply index.bs]: I wonder if the details about the workshop (until "it's worth noting" below) should be moved to an appendix - while very useful, it feels a bit out of place in the flow of the document.

File: index.bs
Comment (dontcallmedom) [inline index.bs:479]: I'm happy to leave this for a future pull request, but I think there is indeed substantive work to harmonize the phrasing of the risks, with possibly merging / sorting some of them as well.

File: index.bs
Comment (dontcallmedom) [inline index.bs]: In terms of mapping mitigations to risks, I would use a naming scheme for risks (e.g. FND-R1 for risk 1 under **F**airness and **N**on-**D**iscrimination), and then references these names from the mitigations that address them.

Comment (anssiko) [reply index.bs]: Done in https://github.com/webmachinelearning/ethical-webmachinelearning/pull/20/commits/b00171e56c8d40f936040d19c877b5fccc972aba

File: index.bs
Comment (anssiko) [reply index.bs]: Moved to appendix. Text changes integrated into https://github.com/webmachinelearning/ethical-webmachinelearning/pull/20/commits/13d34dc490bdea3ce0341c4b39f26eb2000c206e

File: index.bs
Comment (anssiko) [reply index.bs]: Integrated into https://github.com/webmachinelearning/ethical-webmachinelearning/pull/20/commits/13d34dc490bdea3ce0341c4b39f26eb2000c206e

File: index.bs
Comment (anssiko) [reply index.bs]: Opened https://github.com/webmachinelearning/ethical-webmachinelearning/issues/21

**Comment by anssiko** [comment]
Thanks @dontcallmedom for your comments, PR updated.

**Comment by anssiko** [comment]
Gentle ping, PTAL @dontcallmedom.

Your feedback has been addressed. I'd like to merge this after your approval so the WG can advance with its W3C Note publication plan.

**Review by dontcallmedom — APPROVED** [review]

**Comment by anssiko** [comment]
TY @dontcallmedom!

**Pull request merged**
